Code/server/GameServer.cpp

```diff
@@ -264,11 +264,21 @@ void GameServer::BindServerCommands()
         }
     });
 }
-
+    /* Update Info fields from user facing CVARS.*/
 void GameServer::UpdateInfo()
 {
-    // Update Info fields from user facing CVARS.
-    m_info.name = sServerName.c_str();
+    const String cServerName = sServerName.c_str();
+
+    if (cServerName.length() > cm_MaxServerNameLength) 
+    {
```

Comment:
Make cm_maxservernamelength a constexpr size_t global within the source like constexpr size_t kMaxSererNameLength… also make the limit a power of two.

Reply:
fixed